internal/elasticsearch/index/template.go

```diff
@@ -74,28 +74,30 @@ func ResourceTemplate() *schema.Resource {
 			MaxItems:    1,
 			Elem: &schema.Resource{
 				Schema: map[string]*schema.Schema{
-					"aliases": {
-						Description: "Aliases to add.",
+					"alias": {
```

Comment:
I'm assuming this is a breaking change, which we're OK with given the current state of the project, correct?

Reply:
Yeah, the first release was kind of beta and there is no guarantees on backwards-compatibility. Which gives us chance now to make the UX more consistent throughout our resources.

Reply:
@olksdr we still do need to communicate such breaking changes, regardless.
I think we should publish release notes for the next version including the resource / changes made and any of the breaking changes

happy to help put these together.